### PR TITLE
Validate markdown help table nesting

### PR DESCRIPTION
**Pull request by mcbouslog**
Clean-up markdown table nesting issues.

### CODE REVIEW

File: css/markdown-editor.styl [hunk 1]
```diff
@@ -62,6 +62,7 @@
 .markdown-editor-help
   margin: 2em
   table
+    margin: auto
```
Comment (aweiksnar) [inline css/markdown-editor.styl]: Thoughts on adding in `width: 100%` here too? So it'll align the tables vertically and fill the container

Comment (mcbouslog) [inline css/markdown-editor.styl]: Yeah I like that, looks better!  I think I properly updated, fixuped and forced push back, but let me know if not.

Comment (aweiksnar) [inline css/markdown-editor.styl]: :white_check_mark:

**Comment by aweiksnar** [comment]
:surfer: Cool cool, this definitely takes care of those react nesting warnings

**Pull request merged**